site/content/en/docs/concepts/preemption.md

```diff
@@ -54,11 +54,14 @@ Kueue offers two preemption algorithms. The main difference between them is the
 preemptions from a ClusterQueue to others in the Cohort, when the usage of the preempting ClusterQueue is
 already above the nominal quota. The algorithms are:
 
-- **[Classic Preemption](#classic-preemption)**: Preemption in the cohort only happens when the usage of the preempting ClusterQueue
-  will be under the nominal quota after the ongoing admission process, or when all the candidates for preemption belong to
-  the same ClusterQueue as the preempting Workload. In other words, ClusterQueues
-  can only borrow quota from others in the cohort if they do not preempt admitted Workloads from
-  other ClusterQueues. ClusterQueues in a cohort borrow resources in a first-come first-served fashion.
+- **[Classic Preemption](#classic-preemption)**: Preemption in the cohort only happens when:
```

Comment:
at a higher level (not caused by this PR, but a comment on the previous state of the documentation):

We have a whole section for `Classical Preemption` and for `Fair Sharing`. Does it make sense to have the description very high-level here, and go into detail in those respective sections? Or to make those sections subsections of `Preemption algorithms`, rather than top-level sections on their own?

also curious to hear @mimowo's thoughts here.

Reply:
I think the main objective is to make docs clear for readers, and the current structure seems to serve the purpose.  Putting them under the `Preemption algorithms` seems more accurate, but they have some subsections, so I'm not sure how this would impact formatting. I'm leaning to keep the structure as is for this PR.

Reply:
Let's keep it as is for now.